cibuildwheel/windows.py

```diff
@@ -107,6 +106,7 @@ def setup_python(python_configuration: PythonConfiguration, dependency_constrain
     if python_configuration.identifier.startswith('cp'):
         installation_path = install_cpython(python_configuration.version, python_configuration.arch, nuget)
     elif python_configuration.identifier.startswith('pp'):
+        assert python_configuration.url is not None
         installation_path = install_pypy(python_configuration.version, python_configuration.arch, python_configuration.url)
```

Comment:
Really cool that this works!

Reply:
Yes, indeed. Makes perfect sense, but cool feature of mypy type inference :-)